handlers/Careers/components/CareersContent/styles.css

```diff
@@ -2,21 +2,33 @@
 
 .CareersContent {}
 
+.CareersContent-title {
+  color: #F16521;
+  font-size: 1.8em;
+  padding: 1em 0;
+  text-align: center;
+  text-transform: uppercase;
+}
+
 .CareersContent-list {
   list-style: none;
   padding: 0;
 }
 
 .CareersContent-item {
-  border-top: 1px solid #ccc;
+  border-bottom: 1px solid #ccc;
   padding: 1.5em;
   position: relative;
   list-style: none;
 }
 
+.CareersContent-item:last-child {
```

Comment:
likely negligible, but `:first-child` matches faster (since it can match immediately and not have to redraw after every streamed bit of markdown).

This doesn’t need to change to be merged in.